PlatformConfigurations/IEEE802.1xPortBasedAuthentication.md

```diff
@@ -0,0 +1,872 @@
+
+![Seqeunce Diagram of the 802.1x Progression](../Images/ImagesForPlatformConfiguration/802.1x_Progression.png)
+
+In this document, we will be using the following free publicly available software:
+FreeRADIUS, hostapd, and wpa_supplicant to configure an example of IEEE 802.1x and MAC Address Authentication.
+
+## Example Configuration
+
+Consider the following topology:
+
+![IEEE-802.1x-Configuration](../Images/ImagesForPlatformConfiguration/IEEE-802.1x-ConfigurationExample.png)
+
+First, we will guide you through how to configure 802.1x and then
+show how to configure MAC Address Authentication
+
+## Installations
+
+_NOTE: Before any installation, do not forget to use `$ apt-get update` to fetch the latest version of your package lists. Follow this with the command `$ apt-get upgrade` to first review the changes in the latest versions and then replace the old packages by installing the new ones._
+
+### FreeRADIUS Installation
+
+freeRADIUS is a free open source RADIUS server. It is commonly used by Internet Service Providers
+and Telecommunications companies.
+
+If freeRADIUS is not already installed on your device,
+install the `freeradius` utility with `$ apt-get install freeradius`
+
+For our example configuration, we will install `freeradius` by running
+the `$ apt-get install freeradius` command on the **FreeRADIUS-Server**.
+
+On the FreeRADIUS-Server:
+
+```
+root@localhost:~# apt-get install freeradius
+```
+
+### hostapd Installation
+
+hostapd (host acces point daemon) is a user space daemon software enabling a network interface card
+to act as an access point.
+
+If hostapd is not already installed on your device, install the `hostapd` utility with `$ apt-get install hostapd`
+
+For our example configuration, we will install `hostapd` by running
+the `$ apt-get install hostapd` command on the **DENT-Authenticator**.
+
+On the DENT-Authenticator:
+
+```
+root@localhost:~# apt-get install hostapd
+```
+
+### wpa_supplicant Installation
+
+WPA Supplicant is a Wi-Fi Protected Access (WPA) client
+and IEEE 802.1X supplicant. It implements WPA key negotiation
+with a WPA Authenticator and Extensible Authentication Protocol
+(EAP) authentication with an Authentication Server.
+
+If wpa_supplicant is not already installed on your device, install the `wpasupplicant` utility with `$ apt-get install wpa_supplicant`
+
+For our example configuration, we will install`wpasupplicant` by running
+the command `$ apt-get install wpasupplicant` on the **Supplicant**.
+
+On the Supplicant:
+
+```
+root@localhost:~#  apt-get install wpasupplicant
+```
+
+### iproute2 Package Update
+
+Iproute2 Version 5.18 and beyond includes the bridge port lock feature discussed later in this guide.
+This is a key feature used in implementing IEEE 802.1x. If you do not have iproute2
+or the latest features available from the iproute2 package,
+please visit the appendix at the bottom for information on how to change to the latest version.
```

Comment:
instead of "how to change to the latest version", say "install iprout2 vXYZ"

Reply:
I have now updated this paragraph to say "install iproute2-6.9.0":

![image](https://github.com/user-attachments/assets/82ef6dc6-e4a3-4ed5-a957-1e2de71deb03)